internal/controllers/reconciliation/controller.go

```diff
@@ -349,3 +349,10 @@ func isErrMissingNS(err error) bool {
 	}
 	return strings.Contains(err.Error(), "an empty namespace may not be set")
 }
+
+func isErrNoKindMatch(err error) bool {
+	if err == nil {
+		return false
+	}
+	return strings.Contains(err.Error(), "no matches for kind")
```

Comment:
There's no exported helper from k8s errors or api?

Reply:
Nope, and `errors.ReasonForError` returns a zero value 🤷